dbbackup/management/commands/mediabackup.py

```diff
@@ -69,7 +69,7 @@ def handle(self, **options):
         self.filename = options.get("output_filename")
         self.path = options.get("output_path")
         try:
-            self.media_storage = get_storage_class()()
+            self.media_storage = storages['default']
```

Comment:
This change has potential to break some user configurations.

I recommend a creating a dedicated function in `dbbackup.utils` that first tries `get_storage_class()()` and falls back to `storages[django.db.DEFAULT_DB_ALIAS]` on exceptions.